lib/modules/apostrophe-attachments/lib/api.js

```diff
@@ -20,6 +20,13 @@ module.exports = function(self, options) {
   // configure jquery fileupload to submit one per request.)
 
   self.accept = function(req, file, callback) {
+    self.insert(req, file, {permissions: true}, callback);
+  };
+
+  // Extending the functionnality of `accept`. Provide a way to bypass
+  // permissions for backend tasks purposes.
+
+  self.insert = function(req, file, options, callback) {
     var extension = path.extname(file.name);
```

Comment:
Since your goal here is to be parallel with pieces.insert, please support calls with only three arguments, implying an empty options object. See the pieces.insert implementation for the preferred way to structure this.

Once you do that, accept can just be a bc wrapper that invokes this method with three arguments.